tests/network/hotplug.go

```diff
@@ -304,9 +302,25 @@ var _ = SIGDescribe("nic-hotplug", decorators.InPlaceHotplugNICs, func() {
 	})
 })
 
+func verifyHotplug(plugMethod hotplugMethod) {
```

Comment:
How about moving this closer to the `hotplugMethod` definition?

Reply:
done.